ee/docker-compose.ch.hobby.yml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+
+services:
+    db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: posthog
+            POSTGRES_DB: posthog
+            POSTGRES_PASSWORD: posthog
```

Comment:
If we think this will be used by hobbyists for a "production" setup I highly encourage to don't hardcode any secret into the configuration. Few options:

1. use something like docker compose [secrets](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets)
1. use a value (null maybe?) that will make the setup fail so that a user will be forced to edit this docker compose file before running the service

Reply:
Thanks! I think 2 probably makes most sense, it'll be simplest to do and document.

Reply:
Note that our chart has similar problems. Would this be accessible from the outside? I'd love if we made it as 1-click as possible (so would vote against (2)) & add a doc for securing the instance, which folks can skip if they don't care, e.g. I if I'm setting this up for my blog I don't really care about securing it, so not needing to do extra steps would be nice.

Reply:
I think this is a fair stance... but I think we should make it really clear that "its not secure, we know it and why" because we don't want people to get the impression we don't consider or care about security?

I have a follow up question - how do we handle ClickHouse HTTPS in the helm chart - I'm finding it pretty complex  and manual to setup?